populate.sh

```diff
@@ -22,10 +22,12 @@ read
 
 # Delete the old index, swallow failures if it doesn't exist
 curl -s -XDELETE "$ADDRESS/get-together" > /dev/null
+#curl -s -XDELETE "$ADDRESS/get-together-group" > /dev/null
```

Comment:
Could you also remove these commented lines? I suppose they're for testing? Thanks!